src/tools/auth0/client.ts

```diff
@@ -61,10 +73,14 @@ function checkpointPaginator(client, target, name) {
   };
 }
 
-function pagePaginator(client, target, name) {
-  return async function (...args) {
+function pagePaginator(
+  client: Auth0APIClient,
+  target,
```

Comment:
There's still a bit of mystery what this `target` argument is or should be doing. From what I can tell, it's the Auth0 entity manager in the Node Auth0 library, but it's highly complicated to type ourselves as consumers.